### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by tiffeli**
-in line documentation for functions

### CODE REVIEW

File: app/components/Map.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default class Map extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      position: null,
+      // position: null,
```
Comment (jake-shasteen) [inline app/components/Map.js]: This isn't part of the documentation. What does this change do?

File: app/components/FriendListItem.js [hunk 2]
```diff
@@ -10,8 +10,9 @@ export default class FriendListItem extends Component {
   render() {
     const { friend } = this.props;
     // const pic = friend.picture; TODO: Add picture and email
```
Comment (jake-shasteen) [inline app/components/FriendListItem.js]: Remove this

File: app/actions/action_user.js [hunk 3]
```diff
@@ -64,12 +66,15 @@ export const firebase_check = (userCredentials) => {
           userInfo.name = responseData.name;
           userInfo.email = responseData.email;
           userInfo.picture = responseData.picture.data.url;
+
           //pushes all gathereed infor to database
```
Comment (jake-shasteen) [inline app/actions/action_user.js]: Fix typo here

File: app/actions/action_updatePins.js [hunk 4]
```diff
@@ -15,31 +15,40 @@ function setTarget(payload) {
...
           targetRecentlyShared.longitude = sharedPin.longitude;
           targetRecentlyShared.latitude = sharedPin.latitude;
+
           Alert.alert(message, null,
           [
             {text: 'Show me shared pin!', onPress: () => dispatch(setTarget(targetRecentlyShared))},
             {text: 'OK', onPress: () => console.log('OK Pressed')},
           ]);
+
           //once alerted, yet alertedYet to true so it doesn't alert again
```
Comment (jake-shasteen) [inline app/actions/action_updatePins.js]: Fix typo here

File: app/actions/action_updatePins.js [hunk 5]
```diff
@@ -15,31 +15,40 @@ function setTarget(payload) {
...
+    //this listens and filters for pins that are recently shared by friends
     userData.on("child_added", function(snap) {
       var sharedPin = snap.val();
+
+        //alertedYet is a flag in the shared pin obj that will ensure notification will happen once
         if (sharedPin.alertedYet !== null && sharedPin.alertedYet === false) {
           var message = sharedPin.friend.name + " shared a pin with you!";
-          //if user choose to show shared pin
+
+          //set up info for if user choose to go to the newly shared pin
```
Comment (jake-shasteen) [inline app/actions/action_updatePins.js]: Fix grammar here

File: app/components/Map.js (hunk 1 above)
Comment (tiffeli) [inline app/components/Map.js]: I noticed this state.position does not do anything, so I was testing it out. I want to take it out in a new PR that deals with that proptype bug

**Comment by jake-shasteen** [comment]
For each function that you write documentation for, it will be value added to explain what inputs the function is expecting, and what it is outputting or doing as a side effect. For example, if we have a function that accepts a 'coordinates' object as input, we should explain that this refers to an object with longitude and latitude.

For each class or react component that we write documentation for, we should talk about what it does and what props it expects / accepts.

**Comment by tiffeli** [comment]
@jake-shasteen :heavy_check_mark: fixed! Thanks!

**Comment by jake-shasteen** [comment]
Please refactor the pre-class and pre-function comments for those things that we discussed in person ( [referenced here](https://github.com/InterruptedLobster/findAR/pull/125#issuecomment-203569194) ).

**Comment by tiffeli** [comment]
I'm going to close this, since after reading the comments I've decided to do a more extensive documenting. I'll submit another PR when I'm ready!

**Pull request closed without merging**